include/adaflo/sharp_interface.h

```diff
@@ -851,6 +865,12 @@ class MixedLevelSetSolver : public SharpInterfaceSolver
                              navier_stokes_solver.get_dof_handler_u().get_fe().degree +
                                1);
 
+      navier_stokes_solver.solution.zero_out_ghost_values();
```

Comment:
This is something I don't know: Is it okay to `zero_out_ghost_values` before the output is actually written?

Reply:
Yes, it is OK. `DataOut::build_patches()` does the actual work: it translates the vector data to a format that can be used for any output type. But I think the documentation is intentionally misleading^^